qa/cephadm: update staggered upgrade test to use --image when redeploying mgr

adk3798 opened the pull request:
since we now start from pacific rather than octopus, this flag is supported

Signed-off-by: Adam King <adking@redhat.com>





<!--
  - Please give your pull request a title like

      [component]: [short description]

  - Please use this format for each git commit message:

      [component]: [short description]

      [A longer multiline description]

      Fixes: [ticket URL on tracker.ceph.com, create one if necessary]
      Signed-off-by: [Your Name] <[your email]>

    For examples, use "git log".
-->

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [x] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

mgfritch commented on qa/suites/orch/cephadm/upgrade/3-upgrade/staggered.yaml:
```diff
@@ -19,32 +19,34 @@ tasks:
       # doing staggered upgrade requires mgr daemons being on a version that contains the staggered upgrade code
       # until there is a stable version that contains it, we can test by manually upgrading a mgr daemon
       - ceph config set mgr container_image quay.ceph.io/ceph-ci/ceph:$sha1
```
remove this line since it will be set via the `--image` param?

adk3798 commented:
https://pulpito.ceph.com/adking-2022-10-06_23:49:45-orch:cephadm-wip-adk-testing-2022-10-06-1758-distro-default-smithi/7057387
```
2022-10-07T03:28:50.765 INFO:journalctl@ceph.mon.b.smithi188.stdout:Oct 07 03:28:50 smithi188 ceph-mon[91909]: Upgrade aborted - Got unknown service name "rgw.r.z".
2022-10-07T03:28:50.766 INFO:journalctl@ceph.mon.b.smithi188.stdout:                                           Known services are: dict_keys(['alertmanager', 'grafana', 'iscsi.datapool', 'mgr', 'mon', 'node-exporter', 'prometheus'])
2022-10-07T03:28:50.766 INFO:journalctl@ceph.mon.b.smithi188.stdout:                                           Traceback (most recent call last):
2022-10-07T03:28:50.767 INFO:journalctl@ceph.mon.b.smithi188.stdout:                                             File "/usr/share/ceph/mgr/orchestrator/_interface.py", line 125, in wrapper
2022-10-07T03:28:50.767 INFO:journalctl@ceph.mon.b.smithi188.stdout:                                               return OrchResult(f(*args, **kwargs))
2022-10-07T03:28:50.768 INFO:journalctl@ceph.mon.b.smithi188.stdout:                                             File "/usr/share/ceph/mgr/cephadm/module.py", line 3003, in upgrade_start
2022-10-07T03:28:50.768 INFO:journalctl@ceph.mon.b.smithi188.stdout:                                               raise OrchestratorError(f'Upgrade aborted - Got unknown service name "{service}".\n'
2022-10-07T03:28:50.768 INFO:journalctl@ceph.mon.b.smithi188.stdout:                                           orchestrator._interface.OrchestratorError: Upgrade aborted - Got unknown service name "rgw.r.z".
2022-10-07T03:28:50.769 INFO:journalctl@ceph.mon.b.smithi188.stdout:                                           Known services are: dict_keys(['alertmanager', 'grafana', 'iscsi.datapool', 'mgr', 'mon', 'node-exporter', 'prometheus'])
2022-10-07T03:28:50.894 INFO:journalctl@ceph.mon.a.smithi149.stdout:Oct 07 03:28:50 smithi149 ceph-mon[88700]: from='client.44607 -' entity='client.admin' cmd=[{"prefix": "orch upgrade start", "image": "quay.ceph.io/ceph-ci/ceph:c2199385c91e1708fad7968cb12d5501f3cefcbc", "services": "rgw.r.z", "target": ["mon-mgr", ""]}]: dispatch
```
don't think this is caused by this PR but might as well try to fix it here as it's an issue with the staggered upgrade test.

adk3798 commented:
```
unable to load spec for rgw.r: Cannot add RGW: Realm specified but no zone specified
```
https://pulpito.ceph.com/adking-2022-10-08_22:43:30-orch:cephadm-wip-adk-testing-2022-10-08-1137-distro-default-smithi/7058517/

adk3798 commented:
this time 1 of the 2 staggered upgrade instances passed, the other failing with 
```
Error EINVAL: Cannot set values for --daemon-types, --services or --hosts when upgrade already in progress.
```
which makes me think the upgrade resulting from the previous upgrade command had likely failed, although there is no upgrade status output printed in this case I can use to verify this. At the very least there are still issues with the iscsi being deployed in these tests as there are a bunch of tracebacks about it
```
2022-10-10T04:47:39.171 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:38 smithi203 systemd[1]: Started Ceph iscsi.iscsi.a for ef8a568c-4852-11ed-8437-001a4aab830c.
2022-10-10T04:47:40.172 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: --- Logging error ---
2022-10-10T04:47:40.173 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: Traceback (most recent call last):
2022-10-10T04:47:40.173 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib64/python3.6/logging/handlers.py", line 936, in emit
2022-10-10T04:47:40.173 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     self.socket.send(msg)
2022-10-10T04:47:40.173 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: OSError: [Errno 9] Bad file descriptor
2022-10-10T04:47:40.174 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:
2022-10-10T04:47:40.174 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: During handling of the above exception, another exception occurred:
2022-10-10T04:47:40.174 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:
2022-10-10T04:47:40.174 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: Traceback (most recent call last):
2022-10-10T04:47:40.175 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib64/python3.6/logging/handlers.py", line 857, in _connect_unixsocket
2022-10-10T04:47:40.175 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     self.socket.connect(address)
2022-10-10T04:47:40.175 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: FileNotFoundError: [Errno 2] No such file or directory
2022-10-10T04:47:40.175 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:
2022-10-10T04:47:40.175 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: During handling of the above exception, another exception occurred:
2022-10-10T04:47:40.176 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:
2022-10-10T04:47:40.176 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: Traceback (most recent call last):
2022-10-10T04:47:40.176 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib64/python3.6/logging/handlers.py", line 939, in emit
2022-10-10T04:47:40.176 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     self._connect_unixsocket(self.address)
2022-10-10T04:47:40.177 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib64/python3.6/logging/handlers.py", line 868, in _connect_unixsocket
2022-10-10T04:47:40.177 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     self.socket.connect(address)
2022-10-10T04:47:40.177 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: FileNotFoundError: [Errno 2] No such file or directory
2022-10-10T04:47:40.177 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: Call stack:
2022-10-10T04:47:40.178 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib64/python3.6/threading.py", line 884, in _bootstrap
2022-10-10T04:47:40.178 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     self._bootstrap_inner()
2022-10-10T04:47:40.178 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib64/python3.6/threading.py", line 916, in _bootstrap_inner
2022-10-10T04:47:40.178 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     self.run()
2022-10-10T04:47:40.179 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/bin/rbd-target-api", line 2784, in run
2022-10-10T04:47:40.179 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     logger.info("Started the configuration object watcher")
2022-10-10T04:47:40.179 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: Message: 'Started the configuration object watcher'
2022-10-10T04:47:40.179 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: Arguments: ()
2022-10-10T04:47:40.179 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: --- Logging error ---
2022-10-10T04:47:40.180 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: Traceback (most recent call last):
2022-10-10T04:47:40.180 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib64/python3.6/logging/handlers.py", line 936, in emit
2022-10-10T04:47:40.180 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     self.socket.send(msg)
2022-10-10T04:47:40.180 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: OSError: [Errno 9] Bad file descriptor
2022-10-10T04:47:40.181 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:
2022-10-10T04:47:40.181 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: During handling of the above exception, another exception occurred:
2022-10-10T04:47:40.181 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:
2022-10-10T04:47:40.181 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: Traceback (most recent call last):
2022-10-10T04:47:40.182 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib64/python3.6/logging/handlers.py", line 857, in _connect_unixsocket
2022-10-10T04:47:40.182 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     self.socket.connect(address)
2022-10-10T04:47:40.182 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: FileNotFoundError: [Errno 2] No such file or directory
2022-10-10T04:47:40.182 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:
2022-10-10T04:47:40.183 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: During handling of the above exception, another exception occurred:
2022-10-10T04:47:40.183 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:
2022-10-10T04:47:40.183 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: Traceback (most recent call last):
2022-10-10T04:47:40.183 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib64/python3.6/logging/handlers.py", line 939, in emit
2022-10-10T04:47:40.183 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     self._connect_unixsocket(self.address)
2022-10-10T04:47:40.184 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib64/python3.6/logging/handlers.py", line 868, in _connect_unixsocket
2022-10-10T04:47:40.184 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     self.socket.connect(address)
2022-10-10T04:47:40.184 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: FileNotFoundError: [Errno 2] No such file or directory
2022-10-10T04:47:40.184 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: Call stack:
2022-10-10T04:47:40.185 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib64/python3.6/threading.py", line 884, in _bootstrap
2022-10-10T04:47:40.185 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     self._bootstrap_inner()
2022-10-10T04:47:40.185 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib64/python3.6/threading.py", line 916, in _bootstrap_inner
2022-10-10T04:47:40.185 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     self.run()
2022-10-10T04:47:40.186 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/bin/rbd-target-api", line 2786, in run
2022-10-10T04:47:40.186 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     self.interval))
2022-10-10T04:47:40.186 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: Message: 'Checking for config object changes every 1s'
2022-10-10T04:47:40.186 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: Arguments: ()
2022-10-10T04:47:40.187 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: --- Logging error ---
2022-10-10T04:47:40.187 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: Traceback (most recent call last):
2022-10-10T04:47:40.187 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib64/python3.6/logging/handlers.py", line 936, in emit
2022-10-10T04:47:40.187 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     self.socket.send(msg)
2022-10-10T04:47:40.187 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: OSError: [Errno 9] Bad file descriptor
2022-10-10T04:47:40.188 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:
2022-10-10T04:47:40.188 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: During handling of the above exception, another exception occurred:
2022-10-10T04:47:40.188 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:
2022-10-10T04:47:40.188 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: Traceback (most recent call last):
2022-10-10T04:47:40.189 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib64/python3.6/logging/handlers.py", line 857, in _connect_unixsocket
2022-10-10T04:47:40.189 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     self.socket.connect(address)
2022-10-10T04:47:40.189 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: FileNotFoundError: [Errno 2] No such file or directory
2022-10-10T04:47:40.189 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:
2022-10-10T04:47:40.190 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: During handling of the above exception, another exception occurred:
2022-10-10T04:47:40.190 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:
2022-10-10T04:47:40.190 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: Traceback (most recent call last):
2022-10-10T04:47:40.190 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib64/python3.6/logging/handlers.py", line 939, in emit
2022-10-10T04:47:40.191 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     self._connect_unixsocket(self.address)
2022-10-10T04:47:40.191 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib64/python3.6/logging/handlers.py", line 868, in _connect_unixsocket
2022-10-10T04:47:40.191 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     self.socket.connect(address)
2022-10-10T04:47:40.191 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: FileNotFoundError: [Errno 2] No such file or directory
2022-10-10T04:47:40.192 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: Call stack:
2022-10-10T04:47:40.192 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/bin/rbd-target-api", line 2951, in <module>
2022-10-10T04:47:40.192 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     main()
2022-10-10T04:47:40.192 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/bin/rbd-target-api", line 2861, in main
2022-10-10T04:47:40.192 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     osd_state_ok = ceph_gw.osd_blacklist_cleanup()
2022-10-10T04:47:40.193 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib/python3.6/site-packages/ceph_iscsi_config/gateway.py", line 66, in osd_blacklist_cleanup
2022-10-10T04:47:40.193 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     self.logger.info("Processing osd blacklist entries for this node")
2022-10-10T04:47:40.193 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: Message: 'Processing osd blacklist entries for this node'
2022-10-10T04:47:40.193 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: Arguments: ()
2022-10-10T04:47:40.194 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: --- Logging error ---
2022-10-10T04:47:40.194 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: Traceback (most recent call last):
2022-10-10T04:47:40.194 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib64/python3.6/logging/handlers.py", line 936, in emit
2022-10-10T04:47:40.194 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     self.socket.send(msg)
2022-10-10T04:47:40.195 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: OSError: [Errno 9] Bad file descriptor
2022-10-10T04:47:40.195 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:
2022-10-10T04:47:40.195 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: During handling of the above exception, another exception occurred:
2022-10-10T04:47:40.195 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:
2022-10-10T04:47:40.196 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: Traceback (most recent call last):
2022-10-10T04:47:40.196 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib64/python3.6/logging/handlers.py", line 857, in _connect_unixsocket
2022-10-10T04:47:40.196 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     self.socket.connect(address)
2022-10-10T04:47:40.196 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: FileNotFoundError: [Errno 2] No such file or directory
2022-10-10T04:47:40.197 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:
2022-10-10T04:47:40.197 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: During handling of the above exception, another exception occurred:
2022-10-10T04:47:40.197 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:
2022-10-10T04:47:40.197 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: Traceback (most recent call last):
2022-10-10T04:47:40.197 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib64/python3.6/logging/handlers.py", line 939, in emit
2022-10-10T04:47:40.198 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     self._connect_unixsocket(self.address)
2022-10-10T04:47:40.198 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib64/python3.6/logging/handlers.py", line 868, in _connect_unixsocket
2022-10-10T04:47:40.198 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     self.socket.connect(address)
2022-10-10T04:47:40.198 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: FileNotFoundError: [Errno 2] No such file or directory
2022-10-10T04:47:40.199 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: Call stack:
2022-10-10T04:47:40.199 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/bin/rbd-target-api", line 2951, in <module>
2022-10-10T04:47:40.199 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     main()
2022-10-10T04:47:40.199 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/bin/rbd-target-api", line 2861, in main
2022-10-10T04:47:40.200 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     osd_state_ok = ceph_gw.osd_blacklist_cleanup()
2022-10-10T04:47:40.200 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib/python3.6/site-packages/ceph_iscsi_config/gateway.py", line 110, in osd_blacklist_cleanup
2022-10-10T04:47:40.200 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     rm_ok = self.ceph_rm_blacklist(blacklist_entry.split(' ')[0])
2022-10-10T04:47:40.200 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib/python3.6/site-packages/ceph_iscsi_config/gateway.py", line 37, in ceph_rm_blacklist
2022-10-10T04:47:40.200 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     "{}".format(blacklisted_ip))
2022-10-10T04:47:40.201 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: Message: 'Removing blacklisted entry for this host : 172.21.15.203:6833/374599786'
2022-10-10T04:47:40.201 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: Arguments: ()
2022-10-10T04:47:40.201 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: Traceback (most recent call last):
2022-10-10T04:47:40.201 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/bin/rbd-target-api", line 2951, in <module>
2022-10-10T04:47:40.202 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     main()
2022-10-10T04:47:40.202 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/bin/rbd-target-api", line 2861, in main
2022-10-10T04:47:40.202 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     osd_state_ok = ceph_gw.osd_blacklist_cleanup()
2022-10-10T04:47:40.202 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib/python3.6/site-packages/ceph_iscsi_config/gateway.py", line 110, in osd_blacklist_cleanup
2022-10-10T04:47:40.203 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     rm_ok = self.ceph_rm_blacklist(blacklist_entry.split(' ')[0])
2022-10-10T04:47:40.203 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib/python3.6/site-packages/ceph_iscsi_config/gateway.py", line 45, in ceph_rm_blacklist
2022-10-10T04:47:40.203 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     stderr=subprocess.STDOUT, shell=True)
2022-10-10T04:47:40.203 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib64/python3.6/subprocess.py", line 356, in check_output
2022-10-10T04:47:40.203 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     **kwargs).stdout
2022-10-10T04:47:40.204 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:   File "/usr/lib64/python3.6/subprocess.py", line 438, in run
2022-10-10T04:47:40.204 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]:     output=stdout, stderr=stderr)
2022-10-10T04:47:40.204 INFO:journalctl@ceph.iscsi.iscsi.a.smithi203.stdout:Oct 10 04:47:39 smithi203 ceph-ef8a568c-4852-11ed-8437-001a4aab830c-iscsi.iscsi.a[84090]: subprocess.CalledProcessError: Command 'ceph -n client.iscsi.iscsi.a --conf /etc/ceph/ceph.conf osd blacklist rm 172.21.15.203:6833/374599786' returned non-zero exit status 13.
```

adk3798 commented:
https://pulpito.ceph.com/adking-2022-12-02_21:56:44-orch:cephadm-wip-adk-testing-2022-12-02-0932-distro-default-smithi/

Failures tracked by:

- https://tracker.ceph.com/issues/49287
- https://tracker.ceph.com/issues/57755
- https://tracker.ceph.com/issues/57771
- https://tracker.ceph.com/issues/58145
- https://tracker.ceph.com/issues/58173

It's a lot of failures, but other than the the last one, they are all things we were aware of before the run (even the dead mgr-nfs-upgrade job had a `mount.nfs: mounting smithi174:/fake failed, reason given by server: No such file or directory` issue). The last one (https://tracker.ceph.com/issues/58173) as well seems to be a failure on a test setting flags on a pool, and given all the PRs in this run strictly touched mgr modules, I think it's unlikely any of the attached PRs caused the failures. Overall, I think the failures should only block merging on PRs related to nfs due to https://tracker.ceph.com/issues/58145 making it so we can't properly run our nfs related tests currently.

adk3798 commented:
jenkins retest this please
